packages/fxa-auth-server/lib/senders/emails/templates/cadReminder/index.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import { Story, Meta } from '@storybook/html';
+import storybookEmail, { StorybookEmailArgs } from '../../storybook-email';
+// const configURL = require('../../../../../config').get('contentServer.url')
+
+export default {
+  title: 'Emails/Cad Reminder',
```

Comment:
@Yogita98 I think we should follow the name of the actual template here to make scanning for them easier (calling it `cadReminder`).

Also, I think we can DRY up the `args` needed with a helper [like commonPropsWithOverrides](https://github.com/Yogita98/fxa/commit/fa78b5afdf1788555d17e713ec69763442078dfb#diff-721989f388164241c41b2b3cc4cd438e1aa0c53f2fd2866bbbd655ad9285739cR21-R45). It's not too bad without it in `cadReminder` but if there's a template that has 3, 4, 5 stories, it'll be especially handy to change only the properties that differ from story to story and will clean things up.

Plus, adding types for the expected arguments in each template as they're worked on may set us up for some of [Les' wishful thinking](https://github.com/mozilla/fxa/pull/8765#issuecomment-862794694) that he added in the ADR. 🙂